study-builder/studies/rarex/study.conf

```diff
@@ -1191,22 +1191,19 @@
         "type": "SENDGRID_EMAIL",
         "templates": [
           {
-            "emailTemplate": ${emails.finishConsent1wkReminder},
+            "emailTemplate": ${emails.consent1wkReminder},
             "language": "en",
             "isDynamic": false
           }
         ],
         "linkedActivityCode": "CONSENT",
         "pdfAttachments": []
       },
-      "cancelExpr": """user.studies["rarex"].forms["CONSENT"].hasInstance()
```

Comment:
This event is triggered by the creation of the consent, so there's no need to check that we have an instance of it...